payments_service/auth/tests/test_views.py

```diff
@@ -104,6 +106,7 @@ def test_patch_existing_buyer_with_email(self):
 
         buyer_patcher.assert_called_with({
             'email': self.fxa_email,
+            'locale': 'en-US'
```

Comment:
the name of this test should be changed now to something more like `test_patch_existing_buyer_with_updated_info()`